config/rubocop.yml

```diff
@@ -92,3 +92,8 @@ AllCops:
 Style/BlockDelimiters:
   Exclude:
     - "spec/**/*"
+
+# Ruby version
+
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
Hm, not all projects use .ruby-version. Is there a way to make it "use .ruby-version, but fall back to 2.3"?